src/proxy.js

```diff
@@ -24,6 +30,13 @@ proxy.on('proxyReq', (proxyReq, req) => {
     const { eksToken } = req.user;
     proxyReq.setHeader('Authorization', `Bearer ${eksToken.token}`);
 
+    proxyForwardUserAttr.forEach(item => {
+        const parsed = JSON.parse(item);
+        const forwardHeaderName = Object.keys(parsed)[0];
+        const mappedUserAttr = parsed[forwardHeaderName];
+        proxyReq.setHeader(forwardHeaderName, req.user[mappedUserAttr]);
```

Comment:
What's the expected behaviour when `req.user[mappedUserAttr]` is `undefined`? At the very least I think it should log a warning so that you can debug why your headers aren't coming through properly

Reply:
Yeah, you right! Will add some validation.